src/rhsmlib/services/entitlement.py

```diff
@@ -69,18 +76,125 @@ def get_pools(self, pool_subsets=None, matches=None, pool_only=None, match_insta
         if 'consumed' in pool_subsets:
             results['consumed'] = self.get_consumed_product_pools(**options)
         if 'available' in pool_subsets:
-            results['available'] = self.get_available_pools(**options)
+            available = self.get_available_pools(**options)
+            results['available'] = [x._asdict() for x in available]
```

Comment:
Should be fixed now.  I meant for that `_asdict` line to be for the `consumed` entitlements instead of the `available` ones since the results of `get_available_pools` don't really need to be transformed.